docs/technical-docs/zotero.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: default
+title: Zotero
+parent: Technical Documentation
+nav_order: 5
+---
+
+# Zotero Tutorial
+
+An overview of what Zotero does, how to get started, and how to add to it.
+
+
+
+#### TABLE OF CONTENTS
+1. [What is Zotero?](https://ndclab.github.io/wiki/docs/zotero.html#what-is-zotero?)
+2. [Set-Up](https://ndclab.github.io/wiki/docs/zotero.html#set-up)
+3. [Saving Papers](https://ndclab.github.io/wiki/docs/zotero.html#saving-papers)
+4. [Folder Structure](https://ndclab.github.io/wiki/docs/zotero.html#folder-structure)
+5. [Creating and Navigating Tags](https://ndclab.github.io/wiki/docs/zotero.html#creating-and-navigating-tags)
+6. [Quick Tips](https://ndclab.github.io/wiki/docs/zotero.html#quick-tips)
```

Comment:
Great job on linking!  FYI, you don't have to include the full path for internal references like this; you only need the "#" and the text that comes after. (FYI, the way you did it does no harm, I'm just letting you know!)

Reply:
Ah okay, good to know, thank you! I copied the format of the pavlovia.md file, so that's where the full path came from.

Reply:
Oh! One of the very few we haven't yet reviewed/revised! :\ Sorry!